src/main/res/values/strings.xml

```diff
@@ -78,24 +78,24 @@
     <string name="link_copied_to_clipboard">Link copied to clipboard</string>
     <string name="links">Links</string>
     <string name="list_animation">List Animations</string>
-    <string name="list_animation_description">Show list animation on the main page</string>
+    <string name="list_animation_description">Animation the list on the main page</string>
```

Comment:
"Animate the list on main page"

Would be a grammatically correct

Reply:
```suggestion
    <string name="list_animation_description">Animate the list on the main page</string>
```

Reply:
Cool

Reply:
```suggestion
    <string name="list_animation_description">Animate the list on the main page</string>
```